frontend/src/upload.js

```diff
@@ -29,25 +33,48 @@ $(document).ready(function() {
     copyBtn.html('Copy');
   });
 
-  let onChange = event => {
-    const file = event.target.files[0];
-
+  // on file upload by browse or drag & drop
+  window.onUpload = event => {
+    event.preventDefault();
+    let file = '';
+    if (event.type == 'drop') {
+      file = event.dataTransfer.files[0];
```

Comment:
Does this work if I drop multiple files on the drop target from Finder? Or will it only handle the first image?